apps/server/src/modules/tool/external-tool/controller/api-test/tool.api.spec.ts

```diff
@@ -224,7 +224,7 @@ describe('ToolController (API)', () => {
 		});
 	});
 
-	describe('[POST] tools/external-tools', () => {
+	describe('[POST] tools/external-tools/import', () => {
```

Comment:
why?

Reply:
I forgot to add that in my PR so there were 2 describes with the same name